src/components/ProjectsPage/Projects.js

```diff
@@ -68,18 +63,100 @@ const Projects = ({
   const [selectedProjectsState, setSelectedProjectsState] = useState('active')
   const [sortProjectId, setSortProjectId] = useState('byName')
   const [deletingProjects, setDeletingProjects] = useState({})
+  const [jobsFilter, setJobsFilter] = useState({
+    dates: {
+      value: [new Date(moment().add(-1, 'days'))]
+    }
+  })
+  const [workflowsFilter, setWorkflowsFilter] = useState({
+    groupBy: GROUP_BY_WORKFLOW,
+    dates: {
+      value: [new Date(moment().add(-1, 'days'))]
+    },
+    state: STATE_FILTER_ALL_ITEMS
+  })
+  const [scheduledFilter, setScheduledFilter] = useState({
+    dates: {
+      value: [new Date(), new Date(moment().add(1, 'days'))]
+    }
+  })
+  const [loadingState, setLoadingState] = useState({
+    jobs: true,
+    workflows: true,
+    scheduled: true
+  })
```

Comment:
why do set loading state inside filter states above?

Reply:
@mavdryk I use "useMemo" to create the stats config and use it display the data.
if loading state is added to filter state and passed inside the statsConfig, it creates multiple re-renders.
If used from the Filter, then need to pass as props all 3 filters objects. which seems as unnecessary.